xlml/utils/metric.py

```diff
@@ -165,6 +165,23 @@ def download_object_from_gcs(source_location: str, destination_location: str) ->
   logging.info(f"Download JSON Lines file to: {destination_location}")
 
 
+def delete_object_from_gcs(source_location: str) -> None:
```

Comment:
Can we avoid to delete object from `metric.py`? This may affect other tests if we did not config it right.

Reply:
You need to specify `clean_up_gcs=True,` in the job config in order to delete the result object in GCS bucket.

The reason to add this is to prevent `post_process` step from using the result from previous run if the current run fail to generate result.